lib/widgets/payment_details_dialog.dart

```diff
@@ -119,7 +119,7 @@ Future<Null> showPaymentDetailsDialog(
                     child: Scrollbar(
                       child: SingleChildScrollView(
                         child: AutoSizeText(
-                          paymentInfo.description,
+                          '${paymentInfo.description} ${paymentInfo.pending ? "(Waiting to be broadcasted)." : ""}',
```

Comment:
Won't this message show up also for pending lightning payment that is not a part of swap?

Reply:
Fixing. Thank you. I've made a note to test every payment type whenever I make such a change.